source-hello-world/Dockerfile

```diff
@@ -11,19 +9,20 @@ WORKDIR /builder
 COPY go.* ./
 RUN go mod download
 
-# Build the connector projects we depend on.
+COPY go                 ./go
+COPY source-boilerplate ./source-boilerplate
 COPY source-hello-world ./source-hello-world
 
 # Run the unit tests.
 RUN go test -v ./source-hello-world/...
 
 # Build the connector.
-RUN go build -o ./connector -v ./source-hello-world/...
+RUN go build -o ./connector -v ./source-hello-world
 
 
 # Runtime Stage
 ################################################################################
-FROM ${BASE_IMAGE}
+FROM ghcr.io/estuary/base-image:v1
```

Comment:
We do the whole `ARG BASE_IMAGE=...` at the top and then `FROM ${BASE_IMAGE}` thing in all (?) the other connector dockerfiles so it might be best to keep doing that here in case we ever want to change the base image.